report/finished/chapters/concept.tex

```diff
@@ -1,7 +1,31 @@
-\section*{Mechanical structure}
-The final design of the robot. Include CAD renderings etc. 
-\section*{Electrical components}
-Electrical components such as motors, MCUs etc. Why were they chosen
 
-% Putting this here temporarily. Should probably be moved in the future.
-\section*{Requirements}
+\subsection{Mechanical structure}
+The design of the robot is influenced by the LEGO\textregistered Mindstorms\textregistered EV3 set. While the original set has a movable base it was too small and without an arm and manipulator. It also does not incorporate the Dynamixel AX-12A Smart Servos used in the final design so the decision to redesign the robot with design inspiration from the LEGO\textregistered EV3 design. By developing and designing a new platform it gave the possibility to adapt the dimensions and fastenings. It also gave experience in manufacturing methods such as 3D-printing and Laser cutting. The final design is shown in Fig. \ref{fig:concept_rendering}
```

Comment:
too small and without an arm and manipulator. = without a manipulator arm

Reply:
this sentence is a bit unclear; i dont know exactly what its suppose to say. 
 It also does not incorporate the Dynamixel AX-12A Smart Servos used in the final design so the decision to redesign the robot with design inspiration from the LEGO\textregistered EV3 design.